apps/wallet/src/ui/app/components/quests-banner/index.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import ExternalLink from '../external-link';
+import { ampli } from '_src/shared/analytics/ampli';
+import BullsharkOverlay from '_src/ui/assets/images/bullshark-overlay.svg';
+import CapyOverlay from '_src/ui/assets/images/capy-overlay.svg';
+
+export function QuestsBanner() {
```

Comment:
I'm increasingly wanting us to build out an interstitial system that just has these baked as images and is controlled via a feature flag instead of needing to ship code every time.

Reply:
promotions system would definitely be super valuable for these kinds of things